stock-usecase/src/main/java/org/dhis2/android/rtsm/ui/home/screens/HomeScreen.kt

```diff
@@ -41,9 +58,24 @@ fun HomeScreen(
     syncAction: (scope: CoroutineScope, scaffoldState: ScaffoldState) -> Unit = { _, _ -> },
 ) {
     val scaffoldState = rememberScaffoldState()
-    val scope = rememberCoroutineScope()
+
     val dataEntryUiState by manageStockViewModel.dataEntryUiState.collectAsState()
+    val scope = rememberCoroutineScope()
+    var selectedIndex by remember { mutableIntStateOf(0) }
+    val analytics by viewModel.analytics.collectAsState()
```

Comment:
We are using a list of analytics visualizations here just to know if we need to show the bottom bar or not. Why don't we reuse or create a UIState property which handles the home screen layout having there the list of home screens items?

Reply:
That is a good idea I have updated the PR with that.